lease/lessor.go

```diff
@@ -20,22 +20,24 @@ import (
 	"math"
 	"sort"
 	"sync"
-	"sync/atomic"
 	"time"
 
 	"github.com/coreos/etcd/lease/leasepb"
 	"github.com/coreos/etcd/mvcc/backend"
-	"github.com/coreos/etcd/pkg/monotime"
 )
 
 const (
 	// NoLease is a special LeaseID representing the absence of a lease.
 	NoLease = LeaseID(0)
 
-	forever = monotime.Time(math.MaxInt64)
+	// constants from golang/time
+	unixTimeSeconds = int64((1969*365 + 1969/4 - 1969/100 + 1969/400) * (24 * 60 * 60))
+	maxNanoseconds  = 999999999
 )
 
 var (
+	forever = time.Unix(1<<63-1-unixTimeSeconds, maxNanoseconds)
```

Comment:
We shouldn't use `time.Unix` because the `time` returned by `time.Unix` won't include monotonic information as described in 


>**time.Unix**, as well as the unmarshalers t.GobDecode, t.UnmarshalBinary. t.UnmarshalJSON, and t.UnmarshalText always create times with no monotonic clock reading.

[Source](https://golang.org/pkg/time/#hdr-Monotonic_Clocks)

Reply:
@fanminshi You're right. When `forever` variable returned by `time.Unix` doesn't include monotonic information, `time.Sub` used in `Lease.Remaining` calculates in wall time, which may cause an unreasonable phenomenon: remaining time increases itself.

Reply:
@lorneli @heyitsanthony's suggestion using `time.Time{}` as forever sounds like a good idea.

Reply:
@fanminshi I think so too. Changing my commit with @heyitsanthony 's suggestion.